addons/web/static/src/core/notebook/notebook.scss

```diff
@@ -49,7 +49,11 @@
             outline: none;
         }
 
-        &:hover {
+        &:focus-visible, &.active:focus-visible {
+            box-shadow: inset 0 0 $focus-ring-blur $focus-ring-width var(--Notebook__linkFocusRing-color, #{$focus-ring-color});
+        }
+
+        &:hover, &:focus-visible {
```

Comment:
Is it really necessary to define this border-color also for the focus state ?
In Enterprise, this color is already defined with the default state, and it's not really visible on Community since you already have the focus ring :) I suggest to keep it only on hover 👍 

| With | Without |
|--------|--------|
| <img width="168" alt="Capture d’écran 2025-06-18 à 09 14 26" src="https://github.com/user-attachments/assets/7b7fc185-e3f8-458d-b47d-d65e430036e9" /> | <img width="177" alt="Capture d’écran 2025-06-18 à 09 15 48" src="https://github.com/user-attachments/assets/43cd2cfb-24e1-4357-88ed-ec6895a218bb" /> |


|